Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Extract_IP_Indicator.yml

```diff
@@ -0,0 +1,249 @@
+      tags:
+      - ASM
+      script: Builtin|||extractIndicators
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      text:
+        simple: ${inputs.AlertName}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 230
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 7deeb316-2e78-475f-8e86-95eb25925c7e
+    type: regular
+    task:
+      id: 7deeb316-2e78-475f-8e86-95eb25925c7e
+      version: -1
+      name: Create New Indicator
+      description: Change the properties of an indicator
+      script: Builtin|||createNewIndicator
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      tags:
+        simple: ASM - Reported
+      type:
+        simple: IP
+      value:
+        complex:
+          root: ${ExtractedIndicators.IP}
+          filters:
+          - - operator: isNotEmpty
+              left:
+                value:
+                  simple: ${ExtractedIndicators.IP}
+                iscontext: true
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: ca985cec-83d9-4a95-8f7d-f51e5a0aafc8
+    type: condition
+    task:
+      id: ca985cec-83d9-4a95-8f7d-f51e5a0aafc8
+      version: -1
+      name: IP Indicator Found
+      description: Check if a given value exists in the context. Will return 'no'
+        for empty empty arrays. To be used mostly with DQ and selectors.
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "no":
+      - "6"
+      "yes":
+      - "3"
+    scriptarguments:
+      value:
+        complex:
+          root: ${ExtractedIndicators
+          accessor: IP}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 450
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: e7c4ae9f-5b10-4620-8b1d-43d15e2adc6f
+    type: regular
+    task:
+      id: e7c4ae9f-5b10-4620-8b1d-43d15e2adc6f
+      version: -1
+      name: Associate IP Indicator To Alert
+      description: Associate an indicator to a given alert, you can find the indicator
+        to connect either by it's value or it's id
+      script: Builtin|||associateIndicatorToAlert
```

Comment:
@andrew-paloalto  I can't find `associateIndicatorToAlert` automation - is it something you forgot to add or am I missing something?